src/cli/init/index.ts

```diff
@@ -86,13 +86,13 @@ export default function (program: RootCmd) {
             });
 
             // try to open vscode
-            try {
-              execSync("code .", {
-                stdio: "inherit",
-              });
-            } catch (e) {
-              // vscode not found, do nothing
-            }
+            // try {
+            execSync("code .", {
+              stdio: "inherit",
+            });
+            // } catch (e) {
+            // vscode not found, do nothing
+            // }
```

Comment:
We should double-check the error behavior here when `code` is not a binary on the system

Reply:
This is required, otherwise we fail the test and log an exception.